tests/phpunit/integration/tests/Experiments.php

```diff
@@ -39,10 +39,6 @@ public static function wpSetUpBeforeClass( $factory ) {
 		);
 	}
 
-	public static function wpTearDownAfterClass() {
-		self::delete_user( self::$user_id );
-	}
```

Comment:
Why is this removed?

Reply:
WordPress already deletes all users in `\WP_UnitTestCase_Base::tear_down_after_class`

So this is redundant.